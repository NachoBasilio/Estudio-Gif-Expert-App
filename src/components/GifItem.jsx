import PropTypes from 'prop-types';

export default function GifItem({img}) {
  return (
    <div className='card'> 
        <p>{img.title}</p>
        <img src={img.url} alt={img.title} />
     
    </div>
  )
}

GifItem.propTypes = {
    img: PropTypes.object.isRequired
}