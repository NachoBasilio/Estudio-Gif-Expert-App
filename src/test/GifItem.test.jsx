import {render } from '@testing-library/react';
import GifItem  from '../components/GifItem';

describe('Test de screen', () => {
    test('debe de mostrar el componente correctamente', () => {
        const img = {
            title: 'Un titulo',
            url: 'https://localhost/algo.jpg'
        }
        const {container} = render(<GifItem img={img} />);
        expect(container).toMatchSnapshot();
    });
});