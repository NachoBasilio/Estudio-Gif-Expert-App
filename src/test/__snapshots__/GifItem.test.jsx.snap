// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test de screen debe de mostrar el componente correctamente 1`] = `
<div>
  <div
    class="card"
  >
    <p>
      Un titulo
    </p>
    <img
      alt="Un titulo"
      src="https://localhost/algo.jpg"
    />
  </div>
</div>
`;
